lib/Comment.php

```diff
@@ -189,11 +208,18 @@ public function update_depth( $depth = 0 ) {
 		}
 	}
 
+	/**
+	 * Get the depth
+	 *
+	 * @return int
+	 */
 	public function depth() {
 		return $this->_depth;
 	}
 
 	/**
+	 * Checks if is approved
```

Comment:
`Checks if comment is approved.`

Here I added "comment", because the method name `approved()` does’t tell exactly *what* the thing is that is approved.